guides/common/modules/proc_creating-a-personal-access-token.adoc

```diff
@@ -0,0 +1,37 @@
+[id="creating-a-personal-access-token_{context}"]
+= Creating a {PAT}
+
+Use this procedure to create a {PAT}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Users*.
+. Select a user for which you want to create a {PAT}.
+. On the *{PAT}s* tab, click *Add {PAT}*.
+. Enter a *Name* for you {PAT}.
+. Optional: Select the *Expires* date to set an expiration date.
+If you do not set an expiration date, {PAT} will never expire unless revoked.
+. Click Submit.
+You now have the {PAT} available to you on the *{PAT}s* tab.
```

Comment:
```suggestion
```

I am unsure but lean towards dropping this line.

Reply:
I'd prefer to keep this line to prompt the user to look for the PAT as they won't see it again.

Reply:
Maybe reword to "Copy your PAT to your clipboard/Note the PAT because you cannot access it after leaving this page."? Or maybe just drop the admonition and use line 19 as explanation?

Reply:
How about this: I removed the last step and made it part of the admonition.

I imagine that if I was a user and there was not a big **IMPORTANT** admonition striking my eyes, I would definitely lose my first PAT expecting it to stick around forever. (I actually did expect it to stick around forever when testing the procedure.)